CONTRIBUTING.md

```diff
@@ -80,6 +82,15 @@ line of your file_.
 If you want to inherit a language, but don't want the languages inheriting from yours to inherit it,
 you can mark the language as optional (by putting it between parenthesis).
 
+#### Formatting
+
+All queries are expected to follow a standard format, with every node on a single line and indented by two spaces for each level of nesting. You can automatically format the bundled queries by running the provided formatter `./scripts/format-queries.lua` on a single file (ending in `.scm`) or directory to format.
+
+Should you need to preserve a specific format for a node, you can exempt it (and all contained nodes) by placing before it
+```query
```

Comment:
GitHub won't highlight the blocks with `query`, it needs to be `scheme`.

Reply:
But Neovim will ;) And you know which side of _my_ bread is buttered ;)

Reply:
> But Neovim will ;) And you know which side of *my* bread is buttered ;)

Yeah, but we can have *both*.

> (Maybe this will get them to start :))

It won't. :/ (github-linguist/linguist#5746)

Reply:
> Yeah, but we can have both.

No we can't, because the proper highlighting (modelines and all) only works for the `query` language.

Reply:
Better to have 90% in both cases than 100% in one and 0% in the other.

Reply:
The only thing you get on Github is slightly grayer comments; I count that as "90%" ;)